test/cppsuite/tests/cursor_bound_01.cpp

```diff
@@ -51,10 +51,27 @@ class cursor_bound_01 : public test {
     struct bound {
         std::string key;
         bool inclusive;
+        bool lower_bound;
 
         bound() = default;
-        bound(const std::string &_key, bool _inclusive) : key(_key), inclusive(_inclusive) {}
+        bound(int64_t key_size_max, bool _lower_bound) : lower_bound(_lower_bound)
+        {
+            bool set_inclusive = random_generator::instance().generate_integer(0, 1);
+            auto key_size =
+              random_generator::instance().generate_integer(static_cast<int64_t>(1), key_size_max);
+            auto random_key = random_generator::instance().generate_random_string(
+              key_size, characters_type::ALPHABET);
+            key = random_key;
+            inclusive = set_inclusive;
+        }
         ~bound() = default;
+
+        std::string
```

Comment:
You can add `const` at the end of the definition as you are not modifying any member.

Reply:
Tried to do this but realised that we assign `_key = random_key` and that doesn't work 🤔